dali/pipeline/operators/operator.h

```diff
@@ -213,6 +233,100 @@ class Operator : public OperatorBase {
   template <typename B = Backend>
   typename std::enable_if<!std::is_same<B, GPUBackend>::value>::type
   SyncHelper(int /*unused*/, Workspace<B> */*unused*/) {}
+
+  template <typename B = Backend>
+  typename std::enable_if<std::is_same<B, GPUBackend>::value>::type
```

Comment:
maybe simple template specialization would be more readable than doing the enable_if and the is_same above